### PR TITLE
Added TSI retirement notice

### PR DESCRIPTION
**Pull request by dmdenmsft**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by yeskarthik — APPROVED** [review]

**Review by rasidhan — APPROVED** [review]

**Comment by dmdenmsft** [comment]
#sign-off

**Comment by dmdenmsft** [comment]
@Yao725, could you please merge this one?

File: sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Azure Time Series Insights client library for .NET
 
+---
+[//]: <> (This content is similar to https://github.com/MicrosoftDocs/azure-docs/edit/main/includes/tsi-retirement.md)
```
Comment (Yao725) [inline sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/README.md:4]: Can we change this one?
```suggestion
[//]: <> (This content is similar to https://github.com/MicrosoftDocs/azure-docs/blob/main/includes/tsi-retirement.md)
```

**Review by Yao725 — APPROVED** [review]
LGTM

**Pull request merged**